### ISSUE TITLE
 Quaternary Geologic Map of Indiana

### ISSUE THREAD

**Issue by ericjohannes** [UW-Macrostrat/burwell-processing#402]
**Name:**

SURFICIAL_GEOL_MM49_IN: Quaternary Geologic Map of Indiana (Indiana Geological Survey, 1:500,000, Polygon Shapefile)

UNCONSOLIDATED_THICKNESS_250K_IGS_IN.SHP: Unconsolidated Thickness Contours of Indiana (Indiana Geological Survey, 1:250,000, Line Shapefile)

**Source URL:**

SURFICIAL_GEOL_MM49_IN:
https://iu.box.com/s/7nuvm0vp1h8msq9jzx5w2omjnwfqj4u7

UNCONSOLIDATED_THICKNESS_250K_IGS_IN
http://maps.indiana.edu/download/Geology/Surficial_Unconsolidated_Thickness_Contours.zip

**Estimated scale:**

medium

**Folder name:**

**Dataset names:**

SURFICIAL_GEOL_MM49_IN:
data: Surficial_Geol_MM49.gdb/surficial_geol_mm49_in
metadata: Surficial_Geol_MM49_IN.html

UNCONSOLIDATED_THICKNESS_250K_IGS_IN
data: Unconsolidated_Thickness_250k_IGS_IN.shp
metadata: Unconsolidated_Thickness_250k_IGS_IN.txt

**Number of bedrock polygons:**
3505

**Lithology field:**

**Time field:**

b_age, t_age, GEOLOGIC_A

**Stratigraphy name field:**

**Description field:**

DESCRIPTIO

**Comments field:**

DEPOSITION

**Comments:**

This is not a straightforward bedrock geology map so I'm including the abstract here thinking that might help:

SURFICIAL_GEOL_MM49_IN is a polygon shapefile that shows surficial geology in Indiana at a scale of 1:500,000. The mapped units include textural assemblages of quaternary sediments and bedrock lithologies.

UNCONSOLIDATED_THICKNESS_250k_IGS_IN
UNCONSOLIDATED_THICKNESS_250k_IGS_IN.SHP is an Esri line shapefile providing a contoured depiction of the thickness of unconsolidated sediments in Indiana. Thickness contours are represented by polylines (50 foot contour interval) each with a single value that provides estimated thickness of unconsolidated deposits in feet.
the 'Contour (feet)' field has the thickness of the sediment in feet

## Todo:

- [ ] Insert new source record
- [ ] Add data to database
- [ ] Process and add to homogenized tables
- [ ] Process and import lines
- [ ] Process and import points
- [ ] Match
- [ ] Roll tiles

**Comment by aazaff** [comment]
@ericjohannes Is there any way we can get a bottom age for features labeled "Pre-Wisconsonian"?

**Comment by ericjohannes** [comment]
@aazaff just got an answer from a geologist.  I'm told "Pliocene" or the "Tertiary" would work.

**Comment by aazaff** [comment]
Cool, okay so the next step, and this is how all maps you submit from this point forward should be handled, with the `age` column split into a `top_age` column and a `bottom_age` column.

Examples:

Original | b_age | t_age
| ---- | ---- | ---- |
Pliocene | Pliocene | Pliocene
Eocene-Oligocene | Eocene | Oligocene

Please make sure that all maps you submit from now on have done this step.

**Comment by ericjohannes** [comment]
@aazaff can you explain the Pliocene example? Do you mean in cases where our data doesn't contain a top and bottom age we should use the same age value for `top_age` and `bottom_age`?

**Comment by aazaff** [comment]
Exactly, if there is only one age, then that is both the top and the bottom.

**Comment by ericjohannes** [comment]
@aazaff great, thanks for clarifying. Do you need us to add the top and bottom age for this map now or have you done that?

**Comment by aazaff** [comment]
I think doing it for this one would be appropriate. I also wonder if some of the items that I had you merge into #360 might be appropriate to match with this layer instead? It makes sense to keep the surficial/quaternary items together in one map and the bedrock items together in another map?

**Comment by ericjohannes** [comment]
added `b_age` and `t_age` to SURFICIAL_GEOL_MM49_IN's feature class in its gdb. There's a new download link for that.
I rolled the "Unconsolidated Thickness" map in here from https://github.com/UW-Macrostrat/burwell-processing/issues/360 as well

**Comment by cvavrus** [comment]
**Folder name:**
in_quat

Suggested insert:
```
INSERT INTO maps.large (orig_id,source_id,name,age,descrip,t_interval,b_interval,geom) 
SELECT gid,1,deposition,geologic_a,descriptio,late_id,early_id,ST_MakeValid(geom) FROM sources.in_quat where geom is not null;
```
